src/seeders/20220716133454-current-plugs.js

```diff
@@ -4,17 +4,17 @@ module.exports = {
   async up(queryInterface, Sequelize) {
     return await queryInterface.bulkInsert('Plugs', [
       {
-        plugName: 'Type 2 (Mennekes)',
+        plugName: 'typeOne',
         createdAt: new Date(),
         updatedAt: new Date(),
       },
       {
-        plugName: 'Combined Charging System (CCS)',
+        plugName: 'typeTwo',
         createdAt: new Date(),
         updatedAt: new Date(),
       },
       {
-        plugName: 'Charge de Move (CHAdeMO)',
+        plugName: 'typeThree',
         createdAt: new Date(),
         updatedAt: new Date(),
```

Comment:
Are these placeholders or final?

Reply:
Oh I thought it would be easier to use this type of names in the back end. Or maybe even eNum type would be better. But for now just use the simple name so when we look up by name it would be easier and have less problems

Reply:
Yeah I think enum would simplify it, but can come back to it. All good, makes sense to me 👍